var/spack/repos/builtin/packages/py-rsatoolbox/package.py

```diff
@@ -14,21 +14,35 @@ class PyRsatoolbox(PythonPackage):
     git = "https://github.com/rsagroup/rsatoolbox.git"
 
     version("main", branch="main")
+    version("0.1.2", sha256="2d091cbaa33373bf9da4df5ca8d127f0e427431a3db726076090ab2d54fe1213")
+    version("0.1.0", sha256="245f909d31909ba896b765fa51ea019510dd690c6bb8d04b178a9c76ec36dce9")
+    version("0.0.5", sha256="7ede9309755a6173c26f08fd36fa436a11993c7ae0fa9fce05f38be7af0dc6eb")
     version("0.0.4", sha256="84153fa4c686c95f3e83f2cb668b97b82b53dc2a565856db80aa5f8c96d09359")
     version("0.0.3", sha256="9bf6e16d9feadc081f9daaaaab7ef38fc1cd64dd8ef0ccd9f74adb5fe6166649")
 
+    depends_on("python@:3.10", when="@0.0.5:", type=("build", "run"))
```

Comment:
If newer versions don't support 3.11, it's likely that older versions don't either. Also, main supports 3.11, just not 0.1.2.

Reply:
OK, I have changed that.